en/api/cciapi.rst

```diff
@@ -894,6 +900,16 @@ cci_connect_with_url
         --connection URL string when properties(logSlowQueries,slowQueryThresholdMills, logTraceApi, logTraceNetwork) are specified for interface debugging
         URL = "cci:cubrid:192.168.0.1:33000:demodb:::?logSlowQueries=true&slowQueryThresholdMillis=1000&logTraceApi=true&logTraceNetwork=true"
 
+        --connection URL string when useSSL property specified for encrypted connection
+        URL = "cci:cubrid:192.168.0.1:33000:demodb:::?useSSL=true
+
+    .. warning::
+
+        * The **useSSL** flag must match with mode of the broker trying to connect. If the encryption mode is different from the server that trying to connect, that connection request will be rejected. Please refer the following cases that are not allowed.
```

Comment:
1. must match with mode of the broker trying to connect. --> must match with the mode of... ('the' added)

2. If the encryption mode is different from the server that trying to connect, -->  If the encryption mode is different from the server trying to connect, (delete 'the')

3. Please refer the following... --> Please refer to the following... ('to' added)